src/GameOver/GameOver.js

```diff
@@ -20,7 +20,11 @@ const GameOver = (props) => {
     )} 
     if(!props.playerName) {
       return (
-        <p>Error: You have no data, how did you get here?</p>
+        <section className='nes-balloon from-left'>
```

Comment:
Thanks for adding styling to this, you could hardly see it over the space image.